docs/package/gluon-mesh-vpn-wireguard.rst

```diff
@@ -0,0 +1,53 @@
+gluon-mesh-vpn-wireguard
```

Comment:
This document is not referenced anywhere, should probably be added to `index.rst`.

Reply:
ack.